spec.html

```diff
@@ -8602,12 +8608,15 @@ <h1>ModuleNamespaceCreate ( _module_, _exports_ )</h1>
         <emu-alg>
           1. Assert: _module_ is a Module Record.
           1. Assert: _module_.[[Namespace]] is *undefined*.
-          1. Assert: _exports_ is a List of String values.
+          1. Assert: _exports_ is a List of String values or *null*.
```

Comment:
This confused me; do you mean, “exports is either null, or a List of a string values”, or do you mean that the list contains strings or nulls? If the former, maybe using that wording would help.